src/CacheInvalidator.php

```diff
@@ -94,6 +105,8 @@ public function supports($operation)
                 return $this->cache instanceof RefreshInterface;
             case self::INVALIDATE:
                 return $this->cache instanceof BanInterface;
+            case self::TAG:
+                return null !== $this->tagHandler;
```

Comment:
if we deprecate invalidating tags through the CacheInvalidator we should not add this